android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUISlider.java

```diff
@@ -91,11 +91,12 @@ public void processProperties(KrollDict d)
 		} else {
 			maxRange = max;
 		}
-		
 		if (d.containsKey("thumbImage")) {
 			updateThumb(seekBar, d);
 		}
-		
+		if (d.containsKey(TiC.PROPERTY_SPLIT_TRACK)) {
+			seekBar.setSplitTrack(TiConvert.toBoolean(d.get(TiC.PROPERTY_SPLIT_TRACK)));
```

Comment:
Can you guard this line to apply only for API 21 and above? It causes crashes on lower levels since it was added for Lollipop.

https://developer.android.com/reference/android/widget/AbsSeekBar.html#setSplitTrack(boolean)